packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+
+Een bijkomend voordeel van het plaatsen van een label buiten het invoerveld is dat het klik- en tap-oppervlakte automatisch groter is, waardoor het selecteren van een invoerveld voor zowel muis- als touchscreen gebruikers makkelijker is.
+
+#### Positie van labels
+
+Overweeg om labels links boven een invoerveld te plaatsen. Dit biedt meerdere voordelen die bijdragen aan de gebruiksvriendelijkheid.
+
+* Het biedt voldoende ruimte voor labels van verschillende lengtes. Denk bv. ook aan het vertalen van labels (LTR en RTL talen) door vertaalsoftware.
+* Gebruikers die wensen in te zoomen hebben zo de meeste kans om het label corresponderend met de invoer te zien.
+* Het biedt een natuurlijke ‘scan-lijn’ waardoor een formulier automatisch overzichtelijk en consistent wordt.
+
+### Invoertypes
+
+Voor het beantwoorden van vragen bestaan meerdere elementen:
+
+* Tekst invoer
+* Checkboxen
+* Radio buttons
+* Drop-down menu’s
+* Textarea’s
+* Meervoudige keuzelijsten
+* Recaptcha veld
+
+Al deze invoerelementen dienen vergezeld te gaan van een tekst label dat duidelijk de vraag stelt die beantwoord dient te worden.
+
+De verschillende vormen van invoer elementen en hun specifieke richtlijnen vindt je in de componenten-bibliotheek.
+
+Voor ingevulde/al bekende informatie
+
+Als informatie vooraf al bekend is, bijvoorbeeld een adres of telefoonnummer, vul deze dan in maar laat de bezoeker deze informatie wel wijzigen.
+
+Dit kan op twee manieren:
+
+1. Het vooraf invullen van een formulierveld. Hierdoor heeft een gebruiker de mogelijkheid deze informatie direct aan te passen. Dit is uiteraard alleen van toepassing als de bezoeker is ingelogd.
+2. Het tonen van de informatie (_‘read-only’_) met een functie om deze te wijzigen na het klikken op een button.
+
+## Hulp
+
+### Start
+
+Als een gebruiker informatie nodig heeft die zij niet zomaar voor handen zou kunnen hebben, zoals bijvoorbeeld een identiteitsbewijs, voeg dan een overzicht toe wat zij nodig heeft om een formulier volledig in te vullen. Zo komt een gebruiker niet voor verrassingen te staan.
+
+### Placeholder tekst
+
+Optionele placeholder tekst kan helpen bij het bepalen van het juiste gegevensformaat dat voor een bepaalde invoer wordt gevraagd.
+
+Placeholder tekst dient leesbaar te zijn en daarom een juiste contrastverhouding hebben. Omdat gebruikers de placeholder tekst kunnen verwarren met ingevoerde tekst, is het echter raadzaam om placeholder tekst te vermijden.
```

Comment:
ik denk dat we stelliger kunnen zijn over het vermijden van placeholder teksten. Wij zetten immers de patronen voor? 

Ook mis ik een beetje een voorbeeld van een alternatief. Dus we zeggen waar de placeholders voor gebruikt worden , hoe doe je dat in ons systeem waar we placeholders mijden?

Reply:
Ik ben er voor om placeholder tekst te ontmoedigen, dus het is een goed idee om dit anders te verwoorden, ga ik naar kijken!